### ISSUE TITLE
"A world without PDFs" workshop

### ISSUE THREAD

**Issue by InaraFernandez** [cityofaustin/techstack#4568]
Workshopping with departments on how they might consider moving their content beyond PDFs.

**Comment by InaraFernandez** [comment]
@toribr this is just a thought that Jo wanted us to document- let me know if you don't want to be tagged in this type of stuff.

**Comment by srigdon** [comment]
I also think this is a great user research project: how are departments using and what are their needs regarding PDFs? We could combine this with other research goals, too. What do you think, @toribr?

**Comment by toribr** [comment]
@srigdon Yes! Maybe @anukandimalla can reference the content audit, see which departments are using pdfs the most, and then start talking to them about why. Is that what you mean?

**Comment by anukandimalla** [comment]
From one of the Funshop experience - I kind of has this assumption "Who are PDF users"

1.Not sure of using which content type to use and ends up using PDF
2.Content creators who actually like PDF's and don't want to give up PDF
3.Use cases of reports, News releases

I can check existing content audits with the highest use of PDF and we can reach out to identified departments with PDF usage questions as @toribr mentioned.

**Comment by InaraFernandez** [comment]
@anukandimalla I wrote [some notes ](https://docs.google.com/document/d/1p6HzupKaOglgImS7fjCyHlBQM4OoMKHZiqUQOf4RO5o/edit)a while ago on PDF usage using sources from UK.gov and others as well as a conversation I had with Mary Vo from Transportation. I think that department would be a great case study!

**Comment by anukandimalla** [comment]
@InaraFernandez  Here is the list of Departments and PDF count basing on content Audit and Siteimprove
[Link for document](https://docs.google.com/spreadsheets/d/1D3DghHooAt5msY6aSBVVPjZTmWfJAwVZqwX1tMLp_ko/edit?usp=sharing)

**Comment by ablangworthy27** [comment]
@InaraFernandez are you still trying to do this?
